src/components/EntityOpsLevelMaturityProgress.tsx

```diff
@@ -68,18 +68,25 @@ export function EntityOpsLevelMaturityProgress({ levels, serviceLevel }: { level
         >
           <div>
             {levelIcon}
-            <span
+            <div
               style={{
-                textAlign: 'center',
-                position: 'absolute',
-                marginTop: '30px',
-                width: '120px',
-                overflow: 'hidden',
-                textOverflow: 'ellipsis',
+                width: "0px",
+                height: "0px",
+                display: "block",
+                position: "relative",
+                transform: "translate(-60px)"
               }}
             >
-              {level.name}
-            </span>
+              <div style={{
+                width: "120px",
+                textAlign: "center",
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+                whiteSpace: "nowrap"
```

Comment:
Works great! Only one comment: this component **should** show the full level names. Multi-line is fine. Removing these 2 lines does the trick.

```suggestion
```